skbase/tests/conftest.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+"""Common functionality for skbase unit tests."""
+from copy import deepcopy
+from typing import List
+
+from skbase.base import BaseObject
+
+__all__: List[str] = [
+    "FixtureClassParent",
+    "FixtureClassChild",
+    "CompositionDummy",
+    "ResetTester",
+    "InvalidInitSignatureTester",
+    "RequiredParam",
+    "Buggy",
+    "ModifyParam",
+    "NoParamInterface",
+]
+__author__: List[str] = ["RNKuhns"]
```

Comment:
I think some of these are lifted from my code in `sktime.base.test_base`?

Reply:
I believe they are. You moved them to skbase.tests.test_base. I moved them out of there to make them available to other tests in other modules more naturally (in conftest.py).

Reply:
well, so I should be author, is what I mean

Reply:
Yep, good catch. I'll get that updated.